app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PostsController < ApplicationController
+  before_action :logged_in?, only: [:new, :create]
+
+  def index
+    @posts = Post.all
+    @logged_in = logged_in?
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.user_id = current_user.id
```

Comment:
[`Optional`] Here you can assign the `current_user` to the `@post.user` directly. No need to use the ids.